core/arch/arm/include/kernel/mutex.h

```diff
@@ -7,18 +7,26 @@
 
 #include <types_ext.h>
 #include <sys/queue.h>
+#include <kernel/refcount.h>
```

Comment:
minor: maybe move this line and the below one 2 lines above now.

minor: I suggest you update the copyright date here and in **mutex.c**.

Reply:
> minor: I suggest you update the copyright date here and in **mutex.c**.

I'm not sure I can simply change the dates, I think I would have to add a Huawei copyright instead (I'm not contributing as Linaro currently). Does it really matter? Honestly I don't know.